lib/Kubernetes/events/kubernetes_event_monitor.rb

```diff
@@ -0,0 +1,35 @@
+class KubernetesEventMonitor
```

Comment:
The directory for this file should not be capitalized...it should be `lib/kubernetes/...`  I know that many of the other directories in lib are that way, but those are legacy.